src/GuppyDataExplorer/ExplorerFilter/index.jsx

```diff
@@ -38,6 +38,10 @@ class ExplorerFilter extends React.Component {
         }
       }
     }
+    return null;
+  }
+
+  componentDidUpdate() {
```

Comment:
to address a warning found in console while testing using dev.html.
`Warning: ExplorerFilter: getSnapshotBeforeUpdate() should be used with componentDidUpdate(). This component defines getSnapshotBeforeUpdate() only.`